pkg/config/load.go

```diff
@@ -75,6 +75,26 @@ func (p *Prowgen) MergeDefaults(defaults *Prowgen) {
 	p.Rehearsals.DisabledRehearsals = append(p.Rehearsals.DisabledRehearsals, defaults.Rehearsals.DisabledRehearsals...)
 }
 
+func LoadProwgenConfig(folder string) (*Prowgen, error) {
```

Comment:
Nice. As it's the same as [this one](https://github.com/openshift/ci-tools/blob/a3fd3261c807e0da2372cdd6a30f6e0087c578f5/cmd/ci-operator-prowgen/main.go#L104-L121), can you refactor them out?

Reply:
Ah, yes. Nice catch.